judges/quality-of-service/quality-of-service.rb

```diff
@@ -105,3 +105,21 @@ def lifetime(type)
   end
 end
 f.average_backlog_size = issues.empty? ? 0 : issues.inject(&:+) / issues.size
+
+# Rejection PR rate
+all_prs_count = 0
+unmerged_prs_count = 0
+Fbe.unmask_repos.each do |repo|
+  all_q = "repo:#{repo} type:pr closed:>#{$SINCE.utc.iso8601[0..10]}"
```

Comment:
@Yegorov maybe this variable can be inlined. Check this: https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html